scripts/generate-clientset.sh

```diff
@@ -120,7 +123,7 @@ echo "deepcopy"
 "${GOBASE}/bin/deepcopy-gen" \
   ${LOGGING_FLAGS} \
   --input-dirs="${informer_inputs}" \
-  --output-file-base="types.generated" \
+  --output-file-base zz_generated.deepcopy \
```

Comment:
I was expecting this would result in the renaming of a file (`types.generated` -> `zz_generated.deepcopy`), but I don't see any such files. What is this change doing?

Reply:
hmm. you'e right. I was also expecting that but forgot cause it was like a week ago.
I wonder if it didn't regen any because it found them in the package...

Reply:
Ah. It's because they're already named that...

https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/api/configsync/v1alpha1/zz_generated.deepcopy.go

Looks like they've been named that since import.

Reply:
Ah I see, looks like we aren't generating these and checking for a diff in our presubmit checks

Reply:
Been named that since 2020, in fact...